src/fileimpl.h

```diff
@@ -162,7 +162,7 @@ namespace zim
       explicit FileImpl(std::shared_ptr<FileCompound> zimFile);
       FileImpl(std::shared_ptr<FileCompound> zimFile, offset_t offset, zsize_t size);
 
-      std::unique_ptr<IndirectDirentAccessor> getTitleAccessor(const std::string& path);
+      std::unique_ptr<IndirectDirentAccessor> getTitleAccessor(const entry_index_t idx, const std::string& name);
```

Comment:
Looks like you can drop the second parameter and rename this function to `getTitleAccessorV1()`